src/App.tsx

```diff
@@ -1,109 +1,39 @@
 import React from 'react';
 
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import './components/PostInfo/PostInfo.scss';
+import './components/CommentList/CommentList.scss';
+import './components/UserInfo/UserInfo.scss';
+
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
+
+function getUser(id: number): User | null {
```

Comment:
It's small point in checklist: https://github.com/mate-academy/react_static-list-of-posts/blob/master/checklist.md [6 point]
```suggestion
function getUserById(id: number): User | null {
```